broadcast/index.md

```diff
@@ -24,28 +24,91 @@ and let us know there. Otherwise, you can message Manfred Moser or Brian Olsen
 directly on Slack or Twitter.
 
 See us live every two weeks!
+</div>
 
-
-<a class="button" href="/broadcast/episodes.html">See past episodes!</a>
-
+<div markdown="1" class="col-lg-3">
 ## Co-hosts
 
- - Brian Olsen [@bitsondatadev](https://twitter.com/bitsondatadev)
- - Manfred Moser [@simpligility](https://twitter.com/simpligility)
+- Brian Olsen [@bitsondatadev](https://twitter.com/bitsondatadev)
+- Manfred Moser [@simpligility](https://twitter.com/simpligility)
+
+## Watch/Listen
+- Watch:
+<a href="https://www.youtube.com/playlist?list=PLFnr63che7war_NzC7CJQjFuUKLYC7nYh" target="_blank">
+  <i class="fab fa-youtube watch-listen-icon" title="Youtube"></i>
+</a>
+<a href="https://www.twitch.tv/trinodb" target="_blank">
+  <i class="fab fa-twitch watch-listen-icon" title="Twitch"></i>
+</a>
+- Listen:
+<a href="https://podcasts.apple.com/us/podcast/trino-community-broadcast/id1533484786" target="_blank">
+  <i class="fab fa-apple watch-listen-icon" title="Apple"></i>
+</a>
+<a href="https://podcasts.google.com/feed/aHR0cHM6Ly9mZWVkcy5idXp6c3Byb3V0LmNvbS8xMzc0NTMyLnJzcw==" target="_blank">
+  <i class="fab fa-google watch-listen-icon" title="Google"></i>
+</a>
+<a href="https://open.spotify.com/show/53ZrVCCmZSsEmvlNfzpWSL" target="_blank">
+  <i class="fab fa-spotify watch-listen-icon" title="Spotify"></i>
+</a>
```

Comment:
Do we need to do a generic RSS feed or something like that a well.. something that works for any podcast app